src/modules/service-catalog/components/service-catalog-list-item/EmptyState.tsx

```diff
@@ -0,0 +1,63 @@
+import { Anchor, Button } from "@zendeskgarden/react-buttons";
+import { LG, MD } from "@zendeskgarden/react-typography";
+import styled from "styled-components";
+import { useTranslation } from "react-i18next";
+import { getColorV8 } from "@zendeskgarden/react-theming";
+
+const Container = styled.div`
+  padding: ${(p) => p.theme.space.xl} 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: ${(props) => props.theme.space.md};
+`;
+
+const TextContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  gap: ${(props) => props.theme.space.xxs};
+`;
+
+const StyledLink = styled.a`
+  &:visited,
+  &:hover {
+    text-decoration: none;
+    color: ${(props) => getColorV8("grey", 100, props.theme)};
+  }
+`;
+
+export const EmptyState = () => {
+  const { t } = useTranslation();
+  return (
+    <Container>
+      <TextContainer>
+        <LG>
+          {t("service-catalog.empty-state.no-services", "No services in sight")}
+        </LG>
+        <MD>
+          {t(
+            "service-catalog.empty-state.description",
+            "Once services are added to catalog, you’ll find them here."
+          )}
+        </MD>
+        <Anchor isExternal href="https://garden.zendesk.com/" target="_blank">
+          {/* Link will be updated when we will have support article */}
+          {t(
+            "service-catalog.empty-state.support-article-link",
+            "Learn about the service catalog"
+          )}
+        </Anchor>
+      </TextContainer>
+      <Button isPrimary>
+        <StyledLink href="/hc/en-us">
+          {t(
+            "service-catalog.empty-state.go-to-homepage",
+            "Go to the homepage"
+          )}
+        </StyledLink>
+      </Button>
```

Comment:
There is no need to add a link inside a button, and it is also wrong markup.

We can just have an `onClick` handler that redirects the user to the homepage via JS, or style a link as a button. 

We shouldn't redirect the user to `/hc/en-us` but to the current locale. We can just redirect to `/hc` and the backend should redirect to the right locale, or we can use the [page_path helper](https://developer.zendesk.com/api-reference/help_center/help-center-templates/helpers/#page_path-helper) in Curlybar and pass it via props

Reply:
Ok I changed it to button with onClick method and passed helpCenterPath